koji_wrapper/util.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+""" Collection of utility methods  """
+
+
+def convert_to_list(value):
+    """
+    :param value: a value to convert into a string.  Currently, this must be one
+    of [string, list, None]
+    :returns: a flat list version of the value passed in
+    """
+    if isinstance(value, str):
```

Comment:
Do we want to do a isinstance check for list and early return. It is possible to call this with another data type like a dict and value will be returned not a list. Since this is 'public' maybe a call into validate_str_or_list on value is worth while.

Reply:
Perhaps, I will think on this and update Thursday